document/core/exec/instructions.rst

```diff
@@ -1691,7 +1693,11 @@ Table Instructions
    S; F; (\ELEMDROP~x) &\stepto& S'; F; \epsilon
    \end{array}
    \\ \qquad
-     (\iff S' = S \with \SELEMS[F.\AMODULE.\MIELEMS[x]] = \{ \EIELEM~\epsilon \}) \\
+     \begin{array}[t]{@{}r@{~}l@{}}
+     (\iff & F.\AMODULE.\MIELEMS[x] = a \\
+     \wedge & \X{t} = S.\SELEMS[a].\EITYPE \\
+     \wedge & S' = S \with \SELEMS[a] = \{ \EITYPE~t, \EIELEM~\epsilon \}) \\
+     \end{array}
```

Comment:
I think we can actually simplify this to:
```suggestion
     (\iff S' = S \with \SELEMS[F.\AMODULE.\MIELEMS[x]].\EIELEM = \epsilon) \\
```
(and similarly shorten the prose)

Reply:
Done, let me know what you think.